cli/parse-options.js

```diff
@@ -24,23 +21,16 @@ var getDefaults = require('./defaults')
  * passe set the `inMemory: true` option. If it’s not set, leveldown is used
  * with the prefix set to `options.data` + 'data' (`.hoodie/data` by default).
  */
-function parseOptions (options, appOptions, callback) {
-  defaultsDeep(options, appOptions)
-
+function parseOptions (options) {
   var config = {
     loglevel: options.loglevel,
     paths: {
       data: options.data,
       public: options.public
     },
-    connection: {
-      host: options.bindAddress,
-      port: options.port
-    }
```

Comment:
here we parse the CLI options to config that we pass to Hoodie. `connection.host` and `connection.port` are only relevant for hapi, not for Hoodie